packages/phone-number-privacy/signer/test/integration/pnp.test.ts

```diff
@@ -172,6 +187,22 @@ describe('pnp', () => {
         mockOdisPaymentsTotalPaidCUSD.mockReturnValue(onChainBalance)
       })
 
+      it('Should respond with 200 on valid request', async () => {
+        const req = getPnpQuotaRequest(ACCOUNT_ADDRESS1)
+        const authorization = getPnpRequestAuthorization(req, PRIVATE_KEY1)
+
+        const res = await sendRequest(req, authorization, SignerEndpoint.PNP_QUOTA)
+        expect(res.status).toBe(200)
+        expect(res.body).toMatchObject<PnpQuotaResponseSuccess>({
```

Comment:
can you update `toMatchObject` -> `toStrictEqual` in the newly added tests? Otherwise depending on the order of merges, please remind me to changes these in my DB PR :)